libcloud/compute/drivers/ec2.py

```diff
@@ -3608,6 +3626,131 @@ def ex_delete_placement_group(self, name):
         response = self.connection.request(self.path, params=params).object
         return self._get_boolean(response)
 
+    def ex_import_snapshot(self, client_data=None,
+                           client_token=None, description=None,
+                           disk_container=None, dry_run=None, role_name=None):
+        """
+        Imports a disk into an EBS snapshot. More information can be found
+        at https://goo.gl/sbXkYA.
+
+        :param  ClientData: Describes the client specific data (optional)
```

Comment:
You should use underscore notation for all your params. Please fix this across all your docstrings.